src/screens/Rari/RariClaimRgtReview.js

```diff
@@ -65,10 +66,28 @@ const RariClaimRgtReview = ({
   navigation, feeInfo, accountAddress, baseFiatCurrency, rtgPrice,
 }: Props) => {
   const [isSubmitted, setIsSubmitted] = useState(false);
+  const [isFetchingFee, setIsFetchingFee] = useState(false);
+  const [claimFee, setClaimFee] = useState(null);
+
+  useEffect(() => {
+    setIsFetchingFee(true);
+    getClaimRtgFee().then((fee) => {
+      if (fee === null) {
+        Toast.show({
+          message: t('toast.rariFetchDataFailed'),
+          emoji: 'hushed',
+          supportLink: true,
+          autoClose: false,
+        });
+      }
+      setClaimFee(fee);
+      setIsFetchingFee(false);
+    }).catch(() => null);
```

Comment:
Just wondering what happens here from a UX perspective? I think the returning of `null` is what is throwing me off...